pom.xml

```diff
@@ -81,6 +81,8 @@
         <tinkar-starter-data.version>20250813</tinkar-starter-data.version>
         <!-- reasoner options (ElkOwlReasoner, ElkSnomedReasoner, HybridReasoner)-->
         <reasoner-type>ElkSnomedReasoner</reasoner-type>
+        
+        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
Not sure if this works as a property or only as a command line argument. May need to skip in the inherited plugin, e.g.
```
<plugin>
	<groupId>org.apache.maven.plugins</groupId>
	<artifactId>maven-deploy-plugin</artifactId>
	<configuration>
		<skip>true</skip>
	</configuration>
</plugin>
```